crates/cli-support/src/wit/mod.rs

```diff
@@ -484,7 +484,30 @@ impl<'a> Context<'a> {
             Some(class) => {
                 let class = class.to_string();
                 match export.method_kind {
-                    decode::MethodKind::Constructor => AuxExportKind::Constructor(class),
+                    decode::MethodKind::Constructor => {
+                        let msg = "Trying to return JS primitive from constructor, return value will be ignored. Use a builder instead (remove `constructor` attribute).";
```

Comment:
This should include _which_ constructor is causing the error:

```suggestion
                        let msg = format!("The constructor for `{class}` tries to return a JS primitive, which would cause the return value to be ignored. Use a builder instead (remove the `constructor` attribute).");
```

(Don't actually use `format!` though, inline it into the `bail!`.)

Reply:
👍  Done!